app/themes/census/templates/partials/header.html

```diff
@@ -1,12 +1,46 @@
-<header class="header header--census u-mb-s">
-  <div class="u-pt-s u-pb-s" role="banner">
-    <div class="container">
-      <div class="grid grid--gutterless">
-        <div class="grid__col col-6@s">
-          <div class="logo"><img src="{{ url_for('static', filename='img/theme/census/census-logo.svg') }}" alt="2017 Census" /></div>
+{% if language_code  %}
+  {% set languages = language_urls([['cy', 'Cymraeg'], ['en', 'English']], language_code) %}
+{% endif %}
+
+<header class="header u-mb-xs">
+    <div class="header__top" role="banner">
+        <div class="container">
+            <div class="grid grid--gutterless">
+                <div class="grid__col col-6@s">
+                </div>
+                <div class="grid__col col-6@s">
+                    <nav class="nav nav--horizontal  u-d-no--@xs@s u-fr" aria-label="Site services menu">
+                        <ul class="nav__list" role="menubar" aria-label="Site services menu">
+                          {% for language in languages %}
+                              <li class="nav__item" role="menuitem"><a data-qa="switch-language-{{language[0]}}" class="nav__link" href="?language_code={{language[0]}}">{{language[1]}}</a></li>
+                          {% endfor %}
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="header__main" style="background:#6e2585;">
```

Comment:
is this inline style intentional?

Reply:
Nope. We were looking for a way to quickly apply the census colour to the background but decided it doesn't matter for this test.